teos/src/dbm.rs

```diff
@@ -533,7 +532,8 @@ impl DBM {
     /// When a new key is generated, old keys are not overwritten but are not retrievable from the API either.
     pub fn store_tower_key(&self, sk: &SecretKey) -> Result<(), Error> {
         let query = "INSERT INTO keys (key) VALUES (?)";
-        self.store_data(query, params![sk.to_string()])
+        let sk_bytes = Vec::from(sk.secret_bytes());
```

Comment:
It indeed breaks the older storage :(

Reply:
No problem, turns out it can actually be adapted:

 ```
self.store_data(query, params![sk.display_secret().to_string()])
```

Then, `DBM::load_tower_key` will need to be rolled back to load a `String`.